src/models/AdminTask/WhereClauseBuilder/WhereClauseBuilder.ts

```diff
@@ -69,7 +52,7 @@ export const WhereClauseBuilder = {
         conditions.push(`"AdminTask"."${column}" = '${status}'`);
       }
     }
-    return conditions.join(" OR ");
+    return `(${conditions.join(" OR ")})`;
```

Comment:
es realmente necesario? no devuelve un string?

Reply:
pasa que antes se metian los paretesis en el metodo principal. Ahora cada clause mete sus propios parentesis y el metodo principal los joinea con un AND. Lo cambie porque como ahora movi una parte al objeto `TargetWhereClauseBuilder` , entonces quedaba mejor que cada uno pongas sus parentesis